docs/src/maintenance.md

```diff
@@ -0,0 +1,97 @@
+# Optimized Scheduled Maintenance
+_Added in v0.4_
+
+In the real world, some types of resources (notably, fission) require regular scheduled maintenance, which often takes several weeks.
+During this time, the plant produces no power.
+This module allows GenX to find the best time of year for plants to undergo maintenance.
+
+GenX can model scheduled maintenance for only some types of plants:
+
+* Thermal plants with Unit Commitment (THERM=1)
```

Comment:
is this a list with one item? a bit confusing if so because i am expecting to find the remainder of the list somewhere else.